### DISCUSSION TITLE
Character sheet design

### DISCUSSION THREAD

**Discussion by pterodactor3000** [pterodactor3000/dmhub-web#46] [Ideas]
The design can be based on existing character sheet from DMHub.

The idea with it is to lessen eventual confusion when switching from webapp to desktop app, and try to keep consistency throughout the system.

![image](https://user-images.githubusercontent.com/3093486/198824886-850b9ccc-7558-4fb4-a686-3965bea002e1.png)

**Comment by davewx7** [comment]
I think basing it on DMHub's character sheet design is a good choice for consistency. We do have to make it mobile friendly, of course -- which is some good 'scouting' work for what will happen with DMHub's mobile port in the future. :)
